### ISSUE TITLE
scrollbar problem RTL

### ISSUE THREAD

**Issue by farnoosh-wzzx** [olifolkerd/tabulator#3874]
**Describe the bug**
I have a problem in tabulator scrollbar in RTL mode.
as i specified in guide-image, there is a space as much as scrollbar's diameter that it appeared near to the end of the scroll line.
i really stuck in this problem and if you fix the mentioned bug i will appreciate you.

**Tabulator Info**
5.3

**Working Example**
https://jsfiddle.net/vt6qr3mf/7/

**To Reproduce**
just scrolling the tabulator

**Screenshots**
![tabulator-issue](https://user-images.githubusercontent.com/93492121/180759660-96a4b65a-f0df-4492-bc08-bc401f92e8b0.png)

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser: chrome
 - Version : 103.0.5060.114

**Comment by onemoreahmad** [comment]
Try to make the parent div RTL

```
<div dir="rtl">
  <div  id="example-table" style="width:870px"></div>
</div>
```

**Comment by farnoosh-wzzx** [comment]
thanks for your solution, but already i have done it  and it did not works.

**Comment by olifolkerd** [comment]
I have just pushed a fix for this to the 5.4 branch that i am hoping to release in the next week or so.

Cheers

Oli :)
